plugins/amoeba/platforms/cuda/tests/TestCudaAmoebaVdwForce.cpp

```diff
@@ -396,18 +396,41 @@ void testVdwAmmoniaCubicMeanHhg() {
     compareForcesEnergy(testName, expectedEnergy, energy, expectedForces, forces, tolerance);
 }
 
+
+// test VDW w/ sigmaRule=CubicMean and epsilonRule=W-H
+
+void testVdwAmmoniaCubicMeanWH() {
+
+    std::string testName      = "testVdwAmmoniaCubicMeanWH";
+
```

Comment:
Something's clearly wrong in this code.  You begin a function, then go right on to the next one without closing it off, which I guess makes the `testVdwAlchemical` into a local function?

Reply:
You are right. This was caused by my incorrectly deleting the conflict markers.